README.md

```diff
@@ -21,36 +32,38 @@ procedures:
     7. UE Requested PDU Session Release
     8. N/W triggered UE Deregistration
 
-It is also capable to generate and send user data packets (ICMP echo request) 
-and process downlink user data (ICMP echo response) over the established data 
-plane path (N3 Tunnel). 
 
+## Supported System level features
 
-System level features
+    1. It is also capable to generate and send user data packets (ICMP echo request)
```

Comment:
1st statement. so it can start like "Gnbsim can generate and send user data packets"